pkgs/desktops/gnome/extensions/clock-override/default.nix

```
@@ -35,6 +38,6 @@ stdenv.mkDerivation rec {
     license = licenses.mit;
     maintainers = with maintainers; [ rhoriguchi ];
     homepage = "https://github.com/stuartlangridge/gnome-shell-clock-override";
-    broken = versionOlder gnome.gnome-shell.version "3.18";
+    broken = versionOlder gnome.gnome-shell.version "3.40" && versionOlder "40" gnome.gnome-shell.version;
```

At least one of both statements must be wrong :D

```suggestion
    broken = versionOlder gnome.gnome-shell.version "3.40" && versionOlder gnome.gnome-shell.version "40";
```

---

🤦🏻‍♂️